roles/openshift_logging/templates/curator.j2

```diff
@@ -42,10 +42,10 @@ spec:
               value: "{{master_url}}"
             -
               name: "ES_HOST"
-              value: "{{es_host}}"
```

Comment:
this should probably stay es_host so we can change the value that we pass in and change the values of openshift_logging_es_host for subsequent tasks